yarn.lock

```diff
@@ -2231,16 +2231,6 @@ borc@^2.1.2:
     json-text-sequence "~0.1.0"
     readable-stream "^3.6.0"
 
-boxhock_google-finance-data@^0.2.0:
-  version "0.2.0"
-  resolved "https://registry.yarnpkg.com/boxhock_google-finance-data/-/boxhock_google-finance-data-0.2.0.tgz#622cfe59ba7b2eebd9e02c1804cf3d8e50a9b440"
-  integrity sha512-CwFmmwLmuDHBpOz0UfePWGiKTpciKy4leLz4UT4Bhr2B5lObn56gQlKv2EzUGdboDHBPElfhINhzv3u1B/+iXw==
-  dependencies:
-    chrome-aws-lambda "5.3.1"
-    nested-error-stacks "2.1.0"
-    puppeteer "5.3.1"
-    puppeteer-core "5.3.1"
-
```

Comment:
Was this manually removed? I think if we let yarn auto populate, it should also remove some of these dependencies as well.

Reply:
Yes manually, in order not change other parts of yarn.lock. But I can revert it and let this get auto-removed the next time that the .lock file will be updated. Let me know if you want me to revert it.